src/main/java/forestry/plugins/compat/PluginBiomesOPlenty.java

```diff
@@ -171,6 +167,15 @@ private static void addSqueezerRecipes() {
             RecipeManagers.squeezerManager.addRecipe(
                     10, new ItemStack[] {persimmon}, Fluids.JUICE.getFluid(200), mulch, 20);
         }
+
+        Item boPTurnipSeeds = GameRegistry.findItem(BoP, "turnipSeeds");
```

Comment:
There should be a method in the GameRegistry for getting an ItemStack directly, I don't know if it throws a NPE when BoP is not installed though